src/Views/AddDog/AddDog.js

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import DogForm from '../../Comps/DogForm';
+import { addDog } from '../../services/fetchdogs';
+
+
+export default function AddDog() {
+
+  const [img, setImg] = useState('');
+  const [Name, setName] = useState('');
+  const [Age, setAge] = useState('');
+  const [Breed, setBreed] = useState('');
+  const [Desc, setDesc] = useState('');
+  const [error, setError] = useState('');
+  
+  const history = useHistory();
+  
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await addDog({ Name, Age, Desc, Breed, img });
+      alert('Added Good');
```

Comment:
We actually need to have a success message in state per Julie. Super close here!